sympy/physics/control/lti.py

```diff
@@ -249,6 +261,108 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dcgain(self):
+        """
+        Computes the low-frequency gain of the frequency response in the
+        limit as frequency approaches to zero. For systems with pure
+        integrator terms, DC gain is infinite.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 3, s**2 - 9, s)
+        >>> tf1.dcgain()
+        -1/3
+        >>> tf2 = TransferFunction(p**2, p - 3 + p**3, p)
+        >>> tf2.dcgain()
+        0
+        >>> tf3 = TransferFunction(a*p**2 - b, s + b, s)
+        >>> tf3.dcgain()
+        (a*p**2 - b)/b
+
+        """
+        m = Mul(self.num, Pow(self.den, -1, evaluate=False), evaluate=False)
+        return limit(m, self.var, 0)
+
+    def pole(self):
```

Comment:
```suggestion
    def poles(self):
```
seems like a better name, same for `zeros`.

Reply:
[Python-control](https://github.com/python-control/python-control) and MATLAB have the same names. Maybe we should follow the convention?

Reply:
Matlab also has `poles`, which seems to do... exactly the same thing.

I assume matlab avoids `zeros` because it already uses that for creating zero matrices and doesn't have any namespacing.

As far as I can tell, `Python-control` uses `zero(tf)` not `tf.zero()`, so the consistency is already broken.
Making `poles` plural is consistent with `roots` being plural.

Again, will let another maintainer decide.